Gemfile

```diff
@@ -4,13 +4,13 @@ ruby '2.2.3'
 
 group :development do
   gem 'activesupport'
-  gem 'cucumber'
-  gem 'cucumber_lint'
+  gem 'cucumber', '2.1.0'
```

Comment:
We can probably use 2.2.0 but its 2.3.0 that starts using the incompatible gherkin